From fd451f4cfa5225bb4c08db76149d241218d8b528 Mon Sep 17 00:00:00 2001
From: Zhang Fuxin <fxzhang@ict.ac.cn>
Date: Thu, 4 Mar 2021 23:01:27 +0800
Subject: [PATCH 2/4] add a debug information to help debug S3 server related
 failures

---
 model/FullText.inc.php | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/model/FullText.inc.php b/model/FullText.inc.php
index 8eba1d3..f1d1dad 100644
--- a/model/FullText.inc.php
+++ b/model/FullText.inc.php
@@ -147,6 +147,8 @@ class Zotero_FullText {
 				$results->addSuccessful($i, $obj);
 			}
 			catch (Exception $e) {
+				Z_Core::debug($e->getMessage());
+
 				Zotero_DB::rollback();
 				// Clear stale cached version after rollback
 				Zotero_Libraries::refreshUpdatedVersion($libraryID);
-- 
2.34.1

